src/app/components/create-menu/create-menu.module.ts

```diff
@@ -28,10 +28,11 @@ import { CreateMenuComponent } from './create-menu.component';
 import { CommonModule } from '@angular/common';
 import { CoreModule } from '@alfresco/adf-core';
 import { AppToolbarModule } from '../toolbar/toolbar.module';
+import { ToolbarMenuAccessibilityDirective } from './directives/create-menu.accessibility.directive';
```

Comment:
please remove this directive instances and file as well https://github.com/Alfresco/alfresco-content-app/pull/1308/files#diff-8f8bda56550cb22b178ad44951ec013fR37 since we are not using it meaningful